.pre-commit-hooks.yaml

```diff
@@ -65,3 +65,15 @@
     entry: "\uFFFD"
     language: pygrep
     types: [text]
+-   id: ts-no-debugger
+    name: check for debbuger
+    description: 'Prevent use of `debugger` statement'
+    language: pygrep
+    entry: '\bdebugger\b'
+    types: [ts]
+-   id: ts-no-focus-ignore
+    name: check use of focus and ignore of tests
+    description: 'Forbid use of `fit()`, `xit()`, `fdescribe()` and `xdescribe()` test functions'
+    language: pygrep
+    entry: '\b(?:fit|xit|fdescribe|xdescribe)\s*\('
```

Comment:
I'm not sure I agree with `xdescribe` / `xit` -- the equivalent in pytest would be `@pytest.mark.skip` / `@pytest.mark.xfail` which seem fine to check in

I'm also concerned that this isn't generic enough to be helpful for others 🤔

Reply:
Well, as point in favour: one of those made it into the eslint documentation.

Fair enough, so I'll close the PR, considering that the checks can be added with eslint, and I understand that using it would be your preferred way.